server/ServerFSM.cpp

```diff
@@ -652,7 +657,7 @@ sc::result MPLobby::react(const LobbyUpdate& msg) {
         if (psd.m_client_type == Networking::CLIENT_TYPE_HUMAN_OBSERVER ||
             psd.m_client_type == Networking::CLIENT_TYPE_HUMAN_MODERATOR)
         {
-            psd.m_empire_color = GG::Clr(0, 0, 0, 0);
+            psd.m_empire_color = CLR_NONE;
```

Comment:
Should have checked CMakeLists first...server does not link to GiGI.  Propose to leave as is.